### PR TITLE
EST-52: Embed Sales Slide Deck

### PR DESCRIPTION
**Pull request by silviuserdaru**
Create presentations slide deck page.

### CODE REVIEW

File: _css/foundation/_foundation.scss [hunk 1]
```diff
@@ -65,6 +65,8 @@
 @include foundation-orbit;
 @include foundation-pagination;
 @include foundation-progress-bar;
+@include foundation-prototype-sizing;
```
Comment (albertvolkman) [inline _css/foundation/_foundation.scss:68]: Was this on purpose?

Comment (silviuserdaru) [reply _css/foundation/_foundation.scss:68]: @albertvolkman yes, I'm using it already on other task so even if you leave it or not, it is added in another PR... and that is useful for sizing different things, even if I'm not using it on elements on this particular page.

Comment (albertvolkman) [reply _css/foundation/_foundation.scss:68]: Allow the other PR to add it then, please. Remove it here.

Comment (silviuserdaru) [reply _css/foundation/_foundation.scss]: @albertvolkman I removed it.

**Pull request merged**